ltree_hierarchy.gemspec

```diff
@@ -3,29 +3,29 @@ $:.push File.expand_path("../lib", __FILE__)
 require "ltree_hierarchy/version"
 
 Gem::Specification.new do |s|
-  s.name              = "ltree_hierarchy"
-  s.version           = Ltree::Hierarchy::VERSION
-  s.authors           = ["Rob Worley"]
-  s.email             = ["robert.worley@gmail.com"]
-  s.homepage          = "https://github.com/robworley/ltree_hierarchy"
-  s.summary           = "Organize ActiveRecord models into a tree using PostgreSQL's ltree datatype"
-  s.description       = "Organizes ActiveRecord models into a tree/hierarchy using a materialized path implementation based around PostgreSQL's ltree datatype. ltree's operators ensure that queries are fast and easily understood."
+  s.name = "ltree_hierarchy"
+  s.version = Ltree::Hierarchy::VERSION
+  s.authors = ["Rob Worley"]
+  s.email = ["robert.worley@gmail.com"]
+  s.homepage = "https://github.com/robworley/ltree_hierarchy"
```

Comment:
Might be worth changing this authors list and the contact info here as well now?